lib/recurly/base.php

```diff
@@ -133,10 +142,22 @@ protected function __valuesString() {
     return implode($values, ', ');
   }
 
+  public function getHref() {
+    return $this->_href;
+  }
+  // TODO: not sure this should be public
+  public function setHref($href) {
```

Comment:
So just to call out i didn't add this function, it was moved from up above. but yeah i think it needs to be private so that function can call it because one instance can setup another.

Yeah I think it's there so a Recurly_Account gets its href set from `<account href="https://your-subdomain.recurly.com/v2/accounts/1"/>`